.github/workflows/swift-toolchain.yml

```diff
@@ -1718,6 +1726,35 @@ jobs:
       - name: Build Yams
         run: cmake --build ${{ github.workspace }}/BinaryCache/yams
 
+      - name: Configure swift-inspect
+        run: |
+            cmake -B ${{ github.workspace }}/BinaryCache/swift-inspect
+                  -D BUILD_SHARED_LIBS=YES `
+                  -D BUILD_TESTING=NO `
+                  -D CMAKE_BUILD_TYPE=Release `
+                  -D CMAKE_C_COMPILER=cl `
+                  -D CMAKE_C_COMPILER_TARGET=${{ matrix.triple }} `
+                  -D CMAKE_C_FLAGS="${{ needs.context.outputs.CMAKE_C_FLAGS }}" `
+                  -D CMAKE_CXX_COMPILER=cl `
+                  -D CMAKE_CXX_COMPILER_TARGET=${{ matrix.triple }} `
+                  -D CMAKE_CXX_FLAGS="${{ needs.context.outputs.CMAKE_CXX_FLAGS }}" `
+                  -D CMAKE_MT=mt `
+                  -D CMAKE_INSTALL_PREFIX=${{ github.workspace }}/BuildRoot-DevTools/Library/Developer/Toolchains/unknown-Asserts-development.xctoolchain/usr `
+                  -D CMAKE_Swift_COMPILER=${SWIFTC} `
+                  -D CMAKE_Swift_COMPILER_TARGET=${{ matrix.triple }} `
+                  -D CMAKE_Swift_COMPILER_WORKS=YES `
+                  -D CMAKE_Swift_FLAGS="-sdk ${{ github.workspace }}/BuildRoot/Library/Developer/Platforms/Windows.platform/Developer/SDKs/Windows.sdk ${{ needs.context.outputs.CMAKE_Swift_FLAGS }}" `
+                  -D CMAKE_Swift_FLAGS_RELEASE="-O" `
+                  -D CMAKE_SYSTEM_NAME=Windows `
+                  -D CMAKE_SYSTEM_PROCESSOR=${{ matrix.cpu }} `
+                  -G Ninja `
+                  -S ${{ github.workspace }}/SourceCache/swift-llbuild `
+                  -D LLBUILD_SUPPORT_BINDINGS=Swift `
+                  -D SQLite3_LIBRARY=${{ github.workspace }}/BuildRoot/Library/sqlite-3.43.2/usr/lib/SQLite3.lib `
+                  -D SQLite3_INCLUDE_DIR=${{ github.workspace }}/BuildRoot/Library/sqlite-3.43.2/usr/include
+      - name: Build swift-inspect
+        run: cmake --build ${{ github.workspace }}/BinaryCache/swift-inspect
```

Comment:
The swift compiler is not specified, and given that we want to build with the just built swift compiler, this is going to require some additional tweaking.